html/index.html

```diff
@@ -11,15 +11,33 @@
   </head>
 
   <body>
-    <div id="pageTitle">
-      <h1>Binding Of Isaac: Challenge Generator</h1>
+    <div class="wrapper">
+      <div id="pageTitle">
+        <h1>Binding Of Isaac: Challenge Generator</h1>
+      </div>
+      <div class="challengeTitle"><span id="title"></span></div>
+      <div class="container">
+        <div>
+          Challenge #:<br />
+          <span id="seed"></span>
```

Comment:
Could you make the value spans `display: block` so that the `<br />` on the previous line isn't needed?

Reply:
Really good idea, thanks